<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sRecipeManageFDC</class>
 <widget class="QWidget" name="sRecipeManageFDC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="UI_PB_SAVE_RM_FDC">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>48</height>
            </size>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="UI_SA_DEVICE_LIST">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>291</width>
            <height>750</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="UI_LAUOUT_DEVICE_LIST">
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UI_W_RECIPE" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="UI_GB_GROUP_NAME">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>RGA Recipe Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="UI_CB_RECIP_NAME">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>36</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UI_PB_LOAD_RGA_RECIPE">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>Load RGA Recipe</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UI_PB_SAVE_RGA_RECIPE">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>36</height>
               </size>
              </property>
              <property name="text">
               <string>Save RGA Recipe</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QGroupBox" name="groupBox">
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>EQ Recipe List</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QListWidget" name="UI_LW_EQ_RECIPE_LIST">
                <property name="maximumSize">
                 <size>
                  <width>240</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QGroupBox" name="UI_GB_RGA_RECIPE">
             <property name="maximumSize">
              <size>
               <width>700</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string>RGA Recipe</string>
             </property>
             <layout class="QVBoxLayout" name="UI_LAYOUT_RGA_RECIPE">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
